Sources/NIO/SelectableEventLoop.swift

```diff
@@ -222,14 +227,15 @@ internal final class SelectableEventLoop: EventLoop {
     // - see: `EventLoop.execute`
     @inlinable
     internal func execute(_ task: @escaping () -> Void) {
-        self._schedule0(ScheduledTask(task, { error in
+        // nothing we can do if we fail enqueuing here.
+       try? self._schedule0(ScheduledTask(task, { error in
```

Comment:
Can you add a TODO for this to throw in NIO 3?

Reply:
filed as https://github.com/apple/swift-nio/issues/1309